Common/src/main/resources/defaultresources/excavated_variants/excavated_variants/variants/deepercaves.json5

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (C) 2023 Luke Bemish and contributors
+ * SPDX-License-Identifier: LGPL-3.0-or-later
+*/
+
+{
+  mod_id: 'cavesanddepths',
+  provided_stones: [
+    {
+      id: 'chert',
```

Comment:
Stone IDs should be prefixed in case two mods add a stone with the same name -- `cavesanddepths_chert`, for instance. The same goes for stuff throughout the PR.

Reply:
Alright, I guess I can change it.